test/e2e/apimachinery/webhook.go

```diff
@@ -2718,7 +2718,8 @@ func waitWebhookConfigurationReady(ctx context.Context, f *framework.Framework,
 			if strings.Contains(err.Error(), "denied") {
 				return true, nil
 			}
-			return false, err
+			framework.Logf("Unexpected error waiting for webhook configuration to be ready: %v", err)
+			return false, nil
```

Comment:
won't tolerating random errors mask issues with webhooks? our expectation is that webhooks will either intercept and deny, or not intercept and allow

Reply:
Oh, good point. I had suggested we keep the wait going for unexpected errors because it would have prevented this from flaking..  Maybe just drop this from the PR since the below should ensure that the webhook is started and reachable?

Reply:
let me drop the first commit then